Connected/ProjectSettings/ProjectSettings.asset

```diff
@@ -691,7 +691,7 @@ PlayerSettings:
     m_VersionCode: 1
     m_VersionName: 
   apiCompatibilityLevel: 6
-  activeInputHandler: 0
+  activeInputHandler: 1
```

Comment:
This is not appropriate, this should be 0

Reply:
> 

To make sure, is it enough to change the number in settings to 0? Or should something be altered in Unity as well?

Reply:
Yep, that should be enough